packages/salesforce-adapter/src/filters/custom_objects_instances.ts

```diff
@@ -520,6 +520,13 @@ const filterTypesWithManyInstances = async (
   }
 }
 
+const getInaccessibleCustomFields = (objectType: ObjectType): string[] => (
+  Object.entries(objectType.fields)
+    .filter(([, field]) => !isQueryableField(field))
+    .filter(([, field]) => !field.annotations[CORE_ANNOTATIONS.HIDDEN_VALUE])
+    .map(([name]) => name)
```

Comment:
```suggestion
  Object.values(objectType.fields)
    .filter(field) => !isQueryableField(field))
    // No reason to notify users on fields that their value won't be visible
    .filter((field) => !field.annotations[CORE_ANNOTATIONS.HIDDEN_VALUE])
    .map(apiNameSync)
    .filter(isDefined)
```
1. While they're probably the same, it's better to use the **apiName** of a field than the field name in the **ObjectType**.
2. Added a small comment why we filter out hidden value fields

Reply:
Makes sense